src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import Memes from './components/memes/memes';
```

Comment:
- [ ] You have a typo here for memes as well, kindly check it and update it

Reply:
Well resolved ✔️